### DISCUSSION TITLE
Can I use memcpy to compress directory?

### DISCUSSION THREAD

**Discussion by LCUbtuntuCe** [lz4/lz4#1427] [Q&A] [answered]
Can I use memcpy to compress directory?

**Comment by t-mat** [comment]
Hi,

Could you provide more details of your question?  As a verbatim sentence, it seems your question is not directly related to LZ4.

**Comment by LCUbtuntuCe** [comment]
I just see this
<img width="384" alt="无标题" src="https://github.com/lz4/lz4/assets/75744582/bc5a9e1b-3030-4c1a-a33d-c6176eda9c8c">

**Reply by t-mat** [reply, marked as answer]
This row of the table shows `memcpy()` as a control.

It compress and decompress any data with `1.000` compression ratio.
`1.000` means no-(de)compression since it loads and stores data as is.
